Mlem/Models/Trackers/Post Tracker.swift

```diff
@@ -25,53 +25,83 @@ class PostTracker: ObservableObject {
     func loadNextPage(account: SavedAccount, communityId: Int?, sort: SortingOptions?, type: FeedType) async throws {
         defer { isLoading = false }
         isLoading = true
-        
+
         let request = GetPostsRequest(
             account: account,
             communityId: communityId,
             page: page,
             sort: sort,
-            type: type
+            type: type,
+            limit: 50
```

Comment:
based on the comment about trying to make the initial load as fast as possible we could supply a slighter lower value for the first page by doing:
```swift
limit: page == 1 ? 25 : 50
```
It may only save a small fraction of a second on that first load, but the faster the user sees that initial content the better, right?